test_data/README.md

```diff
@@ -0,0 +1,36 @@
+# Firestore Batch Write Items Script
+
+This script allows you to perform a batch write to a Firestore database using a provided JSON file in order to pre-populate a Firestore user with a HouseHomey item collection.
+
+## Prerequisites
+
+Before running the script, make sure you have the following:
+
+### Firebase Service Account Key:
+
+Two options for obtaining, either:
+
+1. Ask @owencooke for the existing project key
+2. Follow the guide [here](https://clemfournier.medium.com/how-to-get-my-firebase-service-account-key-file-f0ec97a21620) to generate a Firebase service account key.
+
+Add the key to the `test_data` directory
+
+## Usage
+
+Install the required Python packages:
+
+```bash
+pip install -r requirements.txt
+```
+
+Run the script with the following command:
+
+```bash
+python3 fb_batch_write.py <username> <path/to/data.json> <path/to/service/account/key.json>
+```
+
+where:
+
```

Comment:
Just a nitpick, we should probably mention in the readme what the format of the JSON file should be (like description, date and cost are mandatory fields). 
Also, do we team members have access to the account key for the project firebase? If yes, where can I access it?

Reply:
Updated the README accordingly. I will share the account key in our Discord server!